COMPONENTS.markdown

```diff
@@ -60,11 +57,11 @@ components:
 - psa-attestation: support code for the Arm PSA Attestation Protocol and
 Token.  This is the attestation protocol that the Veracruz attestation
 service exposes to client code.
-- Sonora: this is the root attestation isolate that is assumed to be present
-on the delegate's machine, and which challenges the authenticity of other
+- SGXRootEnclave: this is the root attestation isolate that is assumed to be present
```

Comment:
If we're following Rust's style guide _strictly_ this should be SgxRootEnclave.

I'm still catching up so if we've decided against this feel free to ignore me. And if we're already using "SGX" in the codebase consistency is preferred to style guides.

Reply:
I tried to use `SgxRootEnclave` throughout, but missed this one.  I'll change this.

Reply:
Ah! That's probably because this is the only one outside of code